test/loader/serializeInstructionTest.cpp

```diff
@@ -1554,4 +1554,127 @@ TEST(SerializeInstructionTest, SerializeDotProductInstruction) {
   Output = {};
   EXPECT_FALSE(Ser.serializeSection(createCodeSec(Instructions), Output));
 }
+
+TEST(SerializeInstructionTest, SerializeEHControlInstruction) {
```

Comment:
Please help to move to the next of tests of control instructions.
It's okay to modify the index number.

Reply:
Sorry l, I don't understand hat do you mean.

Reply:
@q82419

Reply:
Just move it to the next of `SerializeCallControlInstruction`.
Lets put the all tests of controll instructions together.